CHANGES.rst

```diff
@@ -223,6 +223,12 @@ astropy.wcs
 - Added the abstract base class for the low-level WCS API described in APE 14
   (https://doi.org/10.5281/zenodo.1188875). [#7325]
 
+- Added the abstract base class for the high-level WCS API described in APE 14
```

Comment:
Maybe call this two bullet points?  I.e., added the abc is separate from the high-level wrapper.  The latter in particular I think is "the" big-ticket item for users so worth of its own?

Reply:
Done